pyhdb/protocol/lobs.py

```diff
@@ -233,7 +234,7 @@ class Clob(_CharLob):
 
     def __unicode__(self):
         """Convert lob into its unicode format"""
-        return self.data.getvalue().decode(self.encoding)
+        return self.data.getvalue().decode(self.encoding, errors='ignore')
```

Comment:
If someone stores the data then the client must be able to retrieve 100% of the data. In error case it should raise an exception to communicate the problem clearly. Now it would silently ignore errors and problems.
In the future we could think about making that configurable but I actually don't see a good reason why the decoding should fail.